linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel6,rhel7,rhel8,sle15
+prodtype: ocp4,ol7,ol8,rhel6,rhel7,rhel8,sle15
```

Comment:
we're using rhcos4 now as the product for this.

Reply:
ACK'd and done.